### ISSUE TITLE
Social login without Auth0?

### ISSUE THREAD

**Issue by katopz** [scaphold-io/scaphold-issues#6]
Hi,

Auth0 is nice for free dev but it can reach `$745/mo` in prod too.
So the question is do you have other way to so social login?
For example : https://www.reindex.io/docs/security/authentication/#social-login-authentication-providers

Thanks

**Comment by mikeparisstuff** [comment]
Agreed. Auth0 gets steep in price quickly. Options we have considered at oauth.io as well as rolling our own solution as a more long term solution. Any feedback is appreciated.
